client/signup/config/flows-pure.js

```diff
@@ -105,7 +110,6 @@ export function generateFlows( {
 			destination: getSignupDestination,
 			description: 'The improved onboarding flow.',
 			lastModified: '2020-03-03',
-			showRecaptcha: true,
```

Comment:
Sorry, I think I miscommunicated. If the flow already had the prop, then let's keep it. I wanted to say that we needn't add a fresh prop.